webpack.release.config.js

```diff
@@ -21,29 +21,22 @@ module.exports = [{
         // Needed by Cesium for multiline strings
         sourcePrefix: ''
     },
-    amd: {
-        // Enable webpack-friendly use of require in cesium
-        toUrlUndefined: true
-    },
     node: {
         // Resolve node module use of fs
-        fs: "empty"
+        fs: "empty",
+        Buffer: false,
+        http: "empty",
+        https: "empty",
+        zlib: "empty"
     },
     resolve: {
-        alias: {
-            // Cesium module name
-            cesium: path.resolve(__dirname, cesiumSource)
-        }
+        mainFields: ['module', 'main']
     },
     module: {
         rules: [{
             test: /\.css$/,
             use: ['style-loader', {
-                loader: 'css-loader',
-                options: {
-                    // Minify css
-                    minimize: true
```

Comment:
Why is this removed?

Reply:
Because it's no longer supported, I'll have to look into putting it back but I'm not sure it's worth it given that we shop with minimized css already.